src/utils/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const newsApi = axios.create({
+    baseURL: "https://qasims-fe-nc-news-project.onrender.com/api",
+});
+
+export const getArticles = (topic) => {
+    return newsApi.get("/articles", {params: {topic: topic}}).then((res) => {
+            console.log(res)
+            return res.data;
+    })
```

Comment:
Well done on separating out the api call functionality, this is good to see.

Please don't forget to remove the console log though...